src/routes/upload/missionpack/+server.ts

```diff
@@ -4,12 +4,26 @@ import type { RequestEvent } from '@sveltejs/kit';
 
 export async function POST({ request }: RequestEvent) {
 	const pack: MissionPack = await request.json();
```

Comment:
If upload requires a user account, we should check for it on the server side.

Reply:
Please check that I did that right.